action/gen_test.go

```diff
@@ -45,3 +46,30 @@ func TestGenGRPC(t *testing.T) {
 	s := m["proto"].(string)
 	assert.Equal(t, "helloworld.proto", s)
 }
+
+func TestGenDB(t *testing.T) {
```

Comment:
This is a test I added. I am not sure if this is sufficient. Please let me know your thoughts.